logilica_weekly_report/page_settings.py

```diff
@@ -243,17 +243,22 @@ def add_membership_repository(self, repository_slug: str) -> bool:
 
         return False
 
-    def repository_control_button(self, repository_slug: str, *, order=0) -> Locator:
+    def repository_control_button(
+        self, repository_slug: str, *, order=0
+    ) -> Optional[Locator]:
         """Finds repository control button.
 
-        Finds repository control button. Since there might be multiple buttons (up to 2), provides additional logic to pickup the right one.
-        The first button might be found in imported repositories, the other one in available repositories.
+        Finds repository control button. Since there might be multiple buttons
+        (up to 2), provides additional logic to pick up the right one.  The
+        first button might be found in imported repositories, the other one in
+        available repositories.
```

Comment:
interesting that  `docformatter` does not provide these changes for me. Could you have a different (e.g. global) configuration or are you using a specific version (I'm using `1.7.5`)?

Reply:
I made these changes manually.  I'm using `docformatter` `1.7.5` as well, and I was surprised that it didn't (but, I opted not to investigate...).

(Among the mysteries I encountered, some change that I made prompted Black to add a comma to the last parameter in `DashboardPage. download_team_dashboards()`, even though my changes were 10 and 20 lines away.  I went so far as to prove to myself that Black did not add the comma when run on the commit prior to mine, but otherwise I left that mystery unpenetrated as well.)